packages/manager/modules/iam/src/components/identities/selectModal/selectModal.template.html

```diff
@@ -0,0 +1,64 @@
+<div class="modal d-block" tabindex="-1" data-ng-mouseup=":: $ctrl.close()">
+    <div class="modal-dialog ">
+        <div class="modal-content">
+            <oui-modal
+                data-heading="{{ $ctrl.heading }}"
```

Comment:
`heading` is already translated ?
Maybe you can do translation here to avoid making it for each component use
```suggestion
                data-heading="{{ $ctrl.heading | translate }}"
```